hw/top_earlgrey/ip/ast/doc/_index.md

```diff
@@ -1,1060 +1,1178 @@
+additional I/O supplies.
+
+## Core Supplies
+
+The core supplies are generated from the VCC supply. There are two core
+supply domains: VCMAIN and VCAON. VCAON, as its name implies, is the
+always-on core supply used to power components that stay active during
+device low power states. VCMAIN on the other hand, powers most chip
+logic such as RISC-V processor, crypto modules and almost all memories
+and peripherals. The VCMAIN supply can be turned off when requested,
+VCAON on the other hand, is active whenever VCC is active. AST core
+logic is powered by VCAON.
+
+# Power Control and Reset
+
+## Core Power Control and Indication
+
+VCMAIN is the only supply that can be directly influenced by OpenTitan.
+The power manager can request VCMAIN to shutdown through main_pd_ni. The
+state of VCMAIN is reflected by the vcmain_pok_o signal.
+
+## IO Power Indication
+
+IO power state is reflected to OpenTitan by vioa_pok_o and viob_pok_o
+signals
+
+## Main (VCC) Power Detection and Flash Protection
+
+On VCC power-down detection, 'flash_power_ready_h_o', is
+immediately negated. In addition, SYS clock, IO clock and USB clock are
+stopped. This means that negation of the VCC supply always triggers the
+flash brown-out (BOR) protection circuitry.
+
+When entering deep-sleep mode, 'flash_power_down_h_o' is
+asserted before negating VCMAIN until VCMAIN is back up.
+
+## Resets
+
+The AST supports the generation of the root reset for the reset manager.
+It is driven by 'vcaon_pok_o' which is generated inside AST.
+The 'vcaon_pok_o' is activated when the following conditions
+are met: VCC is detected, internal voltage regulator is active and
+'por_ni' reset input is inactive. 'por_ni' is
+driven by an external chip reset pin. The following table and diagrams
+describe the AST sub-modules resets.
+
+| **Components**                                               | **Reset by**             | **Comments**                                                                                                                             |
+|--------------------------------------------------------------|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| Regulators, 'power-OK' logic and always-on clock | self-start / vcaon_pok_o | These circuits come to life shortly after VCC crosses its detection threshold. vcaon_pok_o serves as their register configuration reset. |
+| System/USB/IO clock generators                               | vcmain_pok_o             | vcmain_pok_o is also fed by vcaon_pok_o and por_ni.                                                                                      |
+| Interface functions                                          | Input reset              | Per the corresponding interface [<u>clock domain reset input</u>](#clock-reset-inputs).                                                  |
```

Comment:
The linked subsection `#clock-reset-inputs` does not seem to exist. Did it get out of date?